astro.config.mts

```diff
@@ -0,0 +1,16 @@
+import starlight from '@astrojs/starlight'
+import { defineConfig } from 'astro/config'
+
+// https://astro.build/config
+export default defineConfig({
+  srcDir: './astro',
+  integrations: [
+    starlight({
+      // https://starlight.astro.build/reference/configuration
+      title: 'Ada URL',
+      social: {
+        github: 'https://github.com/ada-url/ada',
```

Comment:
This is for this:

<img width="184" alt="image" src="https://github.com/user-attachments/assets/c4b61272-3a36-4f82-a07d-98c73fbda15e">